nixos/configuration.nix

```diff
@@ -192,6 +200,11 @@ in {
           publicUrl = "https://${siteConfig.serverName}/_irc/";
           ttlSeconds = 0;
         };
+        perRoomConfig = {
+          enabled = true;
+          lineLimitMax = 12;
+          allowUnconnectedMatrixUsers = false; # don't allow overwriting allowUnconnectedMatrixUsers 
+        }
```

Comment:
Also seems fine. I don't actually think allowUnconnectedMatrixUsers makes any difference in practice but it's fine to have it as a matter of least privilege.